ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -192,24 +112,30 @@ The list displays all workflow versions of the selected workflow. Depending on t
 
 ## Workflow editor
 
-*Workflows > Workflows > Tab OVERVIEW > Select workflow > Select version*
+*Process Orchestration > Workflows > Tab OVERVIEW > Select workflow > Select version*
 
 ![Workflow editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
 
-- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)  
-    Click this button to close the workflow editor and return to the list of workflows. All changes are rejected.
+>[Info] Scroll the mouse wheel to zoom in or out. Make sure that the mouse pointer is placed on a workflow element. If necessary, change your scroll wheel settings to adjust the zoom levels.
 
 - ![Points](../../Assets/Icons/Points02.png "[Points]") (Points)  
     Click this button to the right of the workflow name to display the context menu. The following menu entries are available:
 
     - *Triggers*  
         Click this entry to add triggers and conditions. The *Edit trigger for workflow "Workflow name"* window is displayed, see [Edit trigger for workflow "Workflow name"](#edit-trigger-for-workflow-workflow-name).
 
+    - *Show changes*   
+        Click this entry to compare two versions of a workflow to see the differences (insertions/deletions). For detailed information, see [Compare the workflow versions](../Operation/03_CompareWorkflows.md).
+
     - *Import JSON*  
-        Click this entry to import data in JSON format. The *Import JSON* window is displayed, see [Import JSON](#import-json).
+        Click this entry to import a workflow in JSON format. The *Import JSON* window is displayed, see [Import JSON](#import-json).
+        Import the exported JSON code to an existing or new workflow to copy it.
```

Comment:
Ich fand den Satz schwer zu verstehen. Vielleicht "This function allows you to copy..." For detailed information, see ... (Operation/04_CopyWorkflow.md).
Das gilt auch für Export. Alternativ eine [Info] unter Export, wo man diese Information (Import/Export) für beide zusammenfasst und auf Operation/04_CopyWorkflow.md verweist?